core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -50,13 +50,13 @@ THE SOFTWARE.
             ${%No files in directory}
            <j:if test="${showSymlinkWarning}">
              <p>
-                <img id="symlinkalert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" />
+                <img id="symlinkalert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="null"/>
```

Comment:
Since this is showing a symlink warning, I think that the alt text should mention symlink alert
```suggestion
                <img id="symlinkalert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="symlink alert"/>
```

Reply:
This was the wrong suggestion.  Daniel Beck made the correct suggestion in other comments.